modules/apps/translation/translation-web/src/main/resources/META-INF/resources/js/ExportTranslation.js

```diff
@@ -235,19 +235,22 @@ const ExportTranslation = ({
 			onSubmit={(event) => {
 				event.preventDefault();
 
-				const params = {
-					exportMimeType,
-					sourceLanguageId,
-					targetLanguageIds: selectedTargetLanguageIds.join(','),
-				};
+				const url = new URL(exportTranslationURL);
```

Comment:
We can use `addParams` Liferay util instead of raw `searchParams`.

In the previous version, we are avoiding to add `segmentsExperienceIds` empty param but if this check isn't needed by the backend we can do something like that:

```js
event.preventDefault();

location.href = addParams({
  exportMimeType,
  sourceLanguageId,
  segmentsExperienceIds: selectedExperiencesIds.join(','),
  targetLanguageIds: selectedTargetLanguageIds.join(','),
}, exportTranslationURL);
```

Reply:
Thanks, I removed the conditional by mistake.

Reply:
I think is a better option to use our [addParams](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/add_params.js#L35-L39) with a params object to add values to the URL.

And I prefer `if (selectedExperiencesIds.length)` than `if (selectedExperiencesIds.length > 0)` because is less verbose.

They are just subjective ideas about your code feel free to send it like this or update it

Reply:
Or if you prefer the one-line approach but maybe is harder to read

```js
event.preventDefault();

location.href = addParams({
   ...(selectedExperiencesIds.length && {segmentsExperienceIds: selectedExperiencesIds.join(',')}),
  exportMimeType,
  sourceLanguageId,
  targetLanguageIds: selectedTargetLanguageIds.join(','),
}, exportTranslationURL);
```

Reply:
About the length condition, I don't mind; whatever approach is more idiomatic in the current codebase. Do I remove the explicit boolean condition?

About the `addParams` function, the problem is that the inlined version is too "dense" (as you say, difficult to read), and as addParams returns a string (and not the modified URL), we'd need to construct the URL twice to get hold of the params object if we keep the current conditional (which feels a bit wasteful to me).

Reply:
> About the length condition, I don't mind; whatever approach is more idiomatic in the current codebase. Do I remove the explicit boolean condition?

Yep remove it :D

> we'd need to construct the URL twice to get hold of the params object if we keep the current conditional (which feels a bit wasteful to me

I don't think that we need to construct the URL twice, please check if this works for you

```js
const params = {
  exportMimeType,
  sourceLanguageId,
  targetLanguageIds: selectedTargetLanguageIds.join(','),
};

if (selectedExperiencesIds.length) {
  params.selectedTargetLanguageIds = selectedExperiencesIds.join(',');
}

location.href = addParams(params, exportTranslationURL);
```

Reply:
Done.